api/src/test/java/org/apache/iceberg/util/TestCharSequenceMap.java

```diff
@@ -88,20 +92,38 @@ public void testClear() {
     assertThat(map).isEmpty();
   }
 
+  @Test
+  public void testKeySet() {
```

Comment:
Can we add tests that keySets of different CharSequenceMaps match if they are equivalent?